specification.md

```diff
@@ -1274,14 +1281,12 @@ to the trigger/produced event.
 
 #### Error Definition
 
-Error definitions define runtime errors that can occur during workflow execution and how to handle them. For more information,
-see the [Workflow Error Handling section](#Workflow-Error-Handling).
-
 | Parameter | Description | Type | Required |
 | --- | --- | --- | --- |
-| expression | JsonPath expression. Should be evaluated against error data. True if expression result is not empty | string | yes |
-| [errorDataFilter](#error-data-filter) | Error data filter definition | object | yes |
-| [transition](#Transitions) | Next transition of the workflow when `expression` is true | object | yes |
+| error | Domain-specific error name, or '*' to indicate all possible errors | string | yes |
```

Comment:
In case code is defined, can the error be left blank?
"404" would be such an error code when the resource could not be found.

Reply:
i would say no as the error defines the domain specific name. even if on the runtime side only choses to use the error code for the mapping to concrete runtime errors, this name is still going to be used for the reference to the retries top-level definitions

Reply:
AFAICS, there are 3 layers where we can get errors from: 

1. the domain specific naming defined through the API (e.g. my function throws a "OutOfBusinessHoursException"), which would be defined as a response in the Swagger spec, e.g. like a [names component object](https://swagger.io/specification/#components-object)
2. protocol errors of the action; the use of swagger suggests HTTP errors, which are response codes
3. runtime errors, such as transport specific errors (e.g. host not found) or system-related errors (out of memory, stack overflow)